package.json

```diff
@@ -27,6 +27,7 @@
   },
   "homepage": "https://github.com/spinamp/spinamp-sdk",
   "dependencies": {
+    "ethers": "^5.6.9",
```

Comment:
as far as I see we are using only `Signer` type from `ethers`, which means that we are only using it for development.
Please move it to devDependencies, so `ethers` will not be included in bundle

Reply:
well, we depend on just the interface for the signMessage and getAddress functions right? not the whole of ethers? can we include just those? or else just create a simple type/interface for them ourselves

Reply:
But those are just properties of `signer` object which is externally provided

Reply:
yeah exactly what @musnit said. I think it should work when added to `devDependencies` as types are only development thing. It should be included in output index.d.ts files I think. Could you please check it @Geimaj ?

Reply:
updated :+1: